dbcon/mysql/ha_mcs_partition.cpp

```diff
@@ -1184,16 +1318,16 @@ extern "C"
 
             if (partIt->second.status & CPINVALID)
             {
-                if (ct.colWidth <= 8)
+                if (!datatypes::Decimal::isWideDecimalType(ct))
                     output << setw(30) << "N/A" << setw(30) << "N/A";
                 else
-                    output << setw(40) << "N/A" << setw(40) << "N/A";
+                    output << setw(42) << "N/A" << setw(42) << "N/A";
```

Comment:
Could you replace this hardcoded value with a corresponding const from datatypes:: namespace from decimal.h or I can do this after the merge.

Reply:
Done. I've replaced the hard coded value with utils::MAXLENGTH16BYTES